src/gestures/drag/VisualElementDragControls.ts

```diff
@@ -395,27 +385,19 @@ export class VisualElementDragControls {
 
         if (!isDragging) return
 
-        const { dragMomentum, onDragEnd } = this.props
-
-        if (dragMomentum || this.elastic) {
-            const { velocity } = info
-            this.animateDragEnd(velocity)
-        }
+        const { velocity } = info
+        this.animateDragEnd(velocity)
 
-        onDragEnd?.(event, info)
+        this.props.onDragEnd?.(event, info)
```

Comment:
Why not extract this from the props path the top of this function?

Reply:
Never mind, I did not see the full context and that this is the only access to `this.props` that we do